src/sql/platform/capabilities/common/capabilitiesService.ts

```diff
@@ -121,6 +121,11 @@ export interface ConnectionProviderProperties {
 	 * Otherwise ADS will handle the copy request on the UI side.
 	 */
 	supportCopyResultsToClipboard?: boolean;
+
+	/**
+	 * The process ID name used by the Service Host.
```

Comment:
```suggestion
	 * The name to display in the UI for the PID value used to identify connections in the server.
```

Reply:
(update this as needed based on my above comments too if you end up changing the PID name)